receiver/otlpreceiver/internal/logs/otlp.go

```diff
@@ -40,5 +44,18 @@ func (r *Receiver) Export(ctx context.Context, req plogotlp.ExportRequest) (plog
 	err := r.nextConsumer.ConsumeLogs(ctx, ld)
 	r.obsreport.EndLogsOp(ctx, dataFormatProtobuf, numSpans, err)
 
-	return plogotlp.NewExportResponse(), err
+	// Use appropriate status codes for permanent/non-permanent errors
```

Comment:
is there a reference to where this is defined? The comment seems to describe what the code attempts to do, but it'd be great to have a link or reference to how this should be, for people like me who know nothing

Reply:
@atoulme can you check now, I've added a reference.

Reply:
Perfect thanks!